From 09f50d8dbb6b8cbc5aa710cb9c125ae6912a7739 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sat, 14 May 2022 23:15:21 +0800
Subject: [PATCH 57/58] Revert "mgag200 fix memmapsl configuration in GCTL6
 register"

This reverts commit 3aef4df6e1cc4bd5c9ac2b850d4451582dea47e9.
---
 drivers/gpu/drm/mgag200/mgag200_mode.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/mgag200/mgag200_mode.c b/drivers/gpu/drm/mgag200/mgag200_mode.c
index 2a13e297e16df..509968c0d16bc 100644
--- a/drivers/gpu/drm/mgag200/mgag200_mode.c
+++ b/drivers/gpu/drm/mgag200/mgag200_mode.c
@@ -1243,10 +1243,7 @@ static void mgag200_set_format_regs(struct mga_device *mdev,
 	WREG_GFX(3, 0x00);
 	WREG_GFX(4, 0x00);
 	WREG_GFX(5, 0x40);
-	/* GCTL6 should be 0x05, but we configure memmapsl to 0xb8000 (text mode),
-	 * so that it doesn't hang when running kexec/kdump on G200_SE rev42.
-	 */
-	WREG_GFX(6, 0x0d);
+	WREG_GFX(6, 0x05);
 	WREG_GFX(7, 0x0f);
 	WREG_GFX(8, 0x0f);
 
-- 
2.27.0

